app/lib/frontend/dom/dom.dart

```diff
@@ -92,10 +94,14 @@ Node a({
   String? text,
   String? href,
   String? rel,
+  String? target,
   String? title,
 }) {
-  final hasAttributes =
-      attributes != null || href != null || rel != null || title != null;
+  final hasAttributes = attributes != null ||
```

Comment:
Do we really need to compute the `hasAttributes` here? It seems to me that _mergeAttributes handles an empty map just fine...

Also I think the `attributes` parameter could be made into a non-nullable with an empty map as default.

Reply:
I think it is fine either way. IIRC we don't have many places where we emit an empty `<a>...</a>` tag without any attributes, so creating an additional map most of the time doesn't hurt much. But, on the other hand, it doesn't hurt to check either.  Maybe defer this decision a bit, and see what other tags we have and what is the default pattern?